Sources/Client/Model/Channel/Channel.swift

```diff
@@ -128,6 +128,9 @@ public final class Channel: Codable {
     /// Checks for the channel data encoding is empty.
     var isEmpty: Bool { extraData == nil && members.isEmpty && invitedMembers.isEmpty }
     
+    /// Returns the current timestamp. Can be replaced in tests with mock time, if needed.
+    var currentTime: () -> Date = { Date() }
```

Comment:
I like this very much. I thought about how to user `VirtualTime` but didn't find a good solution.